KERNEL.md

```diff
@@ -6,11 +6,11 @@
 
 
 **_Disclaimer:_**<br>
-*This guide is not for the faint of heart! It assumes that you have an idea what a statically linked kernel is and know how to build it for your hardware in some source-based distro.*
+*This guide is not for the faint of heart! It assumes that you have an idea of what a statically linked kernel is and know how to build it for your hardware in some source-based distro.*
 
 ---
 
-This guide implies **IX** package manager in your PATH:
+This guide implies the use of **IX** package manager in your PATH:
```

Comment:
We assume "ix" binary in PATH environment.

Reply:
Corrected.